process_manager/templates/process_manager/index.html

```diff
@@ -28,76 +66,87 @@
-      <div class="card">
-        <div class="card-header">
-          Messages
-          <button id="hide-messages-button"
-                  type="button"
-                  class="btn-close"
-                  aria-label="Hide Messages"
-                  _="on click hide #message-panel show #show-messages-button"></button>
-          <input class="form-control"
-                 type="search"
-                 name="search"
-                 placeholder="Search..."
+  <div class="container-fluid no-padding no-margin">
+    <div class="flex-container" id="main-content">
+      <!-- Process Control Panel -->
+      <div class="process-panel">
+        <div class="card shadow-sm rounded">
+          <div class="card-header bg-primary text-white rounded-top">
+            <h5>Process Control</h5>
+          </div>
+          <div class="card-body">
+            <form method="post" action="{% url 'process_manager:process_action' %}">
+              {% csrf_token %}
+              <div class="d-flex justify-content-between mb-3">
+                <a href="{% url 'process_manager:boot_process' %}"
+                   class="btn btn-primary w-100 me-2">Boot</a>
+                <input type="submit"
+                       value="Restart"
+                       class="btn btn-success w-100 mx-2"
+                       name="action"
+                       onclick="return confirm('Restart selected processes?')"
+                       disabled
+                       _="install disableActionButton">
+                <input type="submit"
+                       value="Flush"
+                       class="btn btn-warning w-100 mx-2"
+                       name="action"
+                       onclick="return confirm('Flush selected processes?')"
+                       disabled
+                       _="install disableActionButton">
+                <input type="submit"
+                       value="Kill"
+                       class="btn btn-danger w-100 ms-2"
+                       name="action"
+                       onclick="return confirm('Kill selected processes?')"
+                       disabled
+                       _="install disableActionButton">
+                <!-- Consistent Show Messages Button -->
+                <button id="show-messages-button"
+                        type="button"
+                        class="btn btn-info w-100 ms-2"
+                        _="on click hide me toggle .hide-messages on #main-content">Show Messages</button>
+              </div>
+              <input class="form-control mb-3"
+                     type="search"
+                     name="search"
+                     placeholder="Search processes..."
+                     hx-get="{% url 'process_manager:process_table' %}"
+                     hx-trigger="input changed delay:500ms, every 1s"
+                     hx-target="div.table-container">
+              <div id="table-container" class="table-container">
```

Comment:
The tables library already wraps the table in a separator div with `class="table-container"` so no need to apply here as well.